src/inputs/Switch.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <label :class="switchClass">
+        <span>{{ this.label }}</span>
+        <div 
+            :name="name"
+            v-on:click="e => this.onChange(!inputValue)"
+        />
+        <field-error :errors="this.validationErrors" />
+    </label>
+</template>
+
+<script>
+    import Vue from 'vue';
+    import _ from 'lodash';
+    import inputProps from '../inputProps';
+    import { validationMixin } from '../validationHelper';
+
+    export default {
+        name: 'SuperSwitch',
+        mixins: [validationMixin],
```

Comment:
This component should also have the `formSchemaMixin`, like all other components.